arch/arm/src/cmake/gcc.cmake

```diff
@@ -66,7 +66,9 @@ execute_process(COMMAND ${CMAKE_C_COMPILER} --version
                 OUTPUT_VARIABLE GCC_VERSION_OUTPUT)
 string(REGEX MATCH "\\+\\+.* ([0-9]+)\\.[0-9]+" GCC_VERSION_REGEX
              "${GCC_VERSION_OUTPUT}")
-set(GCCVER ${CMAKE_MATCH_1})
+set(GCCVER
+    ${CMAKE_MATCH_1}
+    PARENT_SCOPE)
 
 if(GCCVER EQUAL 12)
```

Comment:
Should this also be greater equal?

Reply:
ok, please review https://github.com/apache/nuttx/pull/14241